lang/en.xml

```diff
@@ -408,7 +408,7 @@ Please download and install it.</no_font_exist>
 			<select_bin_rif>Select work.bin/rif</select_bin_rif>
 			<successed_install_license>Successfully installed license.</successed_install_license>
 			<delete_bin_rif>Delete the work.bin/rif file?</delete_bin_rif>
-			<check_bin_rif>Please check correct the work.bin/rif file.</check_bin_rif>
+			<check_bin_rif>Please check the work.bin/rif file.</check_bin_rif>
```

Comment:
complement zrif key, and also you need change lang/state.h

Reply:
wdym by "complement"?